src/bindings/python/fluxacct/accounting/job_archive_interface.py

```diff
@@ -127,14 +122,17 @@ def add_job_records(rows):
     job_records = []
 
     for row in rows:
+        rdict = json.loads(row[6])  # fetch R
```

Comment:
Is `rdict` unused?

Reply:
Ah, you are absolutely right, `rdict` is unused. I must have forgotten to remove it when I found out I could just load R as a string to `rset` instead of converting it to a dictionary first; sorry about that, and thanks for pointing it out. I'll push up a fix.

Reply:
No problem! It surprised me at first that the `json` module was required to use `ResourceSet`. thanks!